src/Domain/DonationStatus.php

```diff
@@ -5,29 +5,102 @@
 /**
  * @link https://docs.google.com/document/d/11ukX2jOxConiVT3BhzbUKzLfSybG8eie7MX0b0kG89U/edit?usp=sharing
  * @todo Consider vs. Stripe options
- *
- * @todo before merging BG2-2296 - check that the prod DB doesn't have any statuses recorded not listed below.
```

Comment:
I did do this before merging the previous PR, forgot to delete the todo earlier.